test/distributed/fsdp/test_fsdp_tp_integration.py

```diff
@@ -8,18 +8,12 @@
 from torch import distributed as dist
 from torch.distributed._shard.sharded_tensor.api import ShardedTensor
 from torch.distributed._shard.sharding_spec import ChunkShardingSpec
-from torch.distributed._tensor import DeviceMesh, DTensor as DT
+from torch.distributed._tensor import DeviceMesh, DTensor as DT, init_device_mesh
 from torch.distributed.fsdp.fully_sharded_data_parallel import (
     CPUOffload,
     FullyShardedDataParallel as FSDP,
-    StateDictType,
 )
-from torch.distributed.tensor.parallel import (
-    PairwiseParallel,
-    parallelize_module,
-    SequenceParallel,
-)
-from torch.distributed.tensor.parallel.fsdp import enable_2d_with_fsdp
+from torch.distributed.tensor.parallel import parallelize_module, SequenceParallel
```

Comment:
not necessary in this PR, we want to remove `SequenceParallel` down the road...

Reply:
Ye. Had a TODO listed in the PR and planned to do it in a follow-up PR